scantext/templates/scantext/includes/license_report.html

```diff
@@ -0,0 +1,2 @@
+<a class="has-text-danger" href="https://github.com/nexB/scancode.io/issues/new?labels=bug&title=License+detection+error+as+`{{ license.license_expression|pprint }}`
+&body=Detection+level+details%0A```python%0A{%0A%20%20%20%20score+:+{{ license.score }}+%0A%20%20%20%20start_line+:+{{ license.start_line }}+%0A%20%20%20%20end_line+:+{{ license.end_line }}+%0A%20%20%20%20matched_length+:+{{ license.matched_length }}+%0A%20%20%20%20match_coverage+:+{{ license.match_coverage }}+%0A%20%20%20%20rule_identifier+:+{{ license.rule_identifier }}%0A}%0A```+%0A%0AMatched+Text%0A```%0A{{ license.matched_text }}%0A```+%0A%0AInput+Text%0A```%0A{{ license.matched_text }}%0A```" target="_blank">Report on Github</a>
```

Comment:
> Could you point me to the place were the design for the reporting to GitHub feature was discussed?

@tdruez here it is

Reply:
This is not a "design discussion", this is code...

Reply:
Reporting license details should be done at `scancode-toolkit issues` page.

Reply:
The project idea says 
```
It will also allow the integrated reporting of license detection issues in the app based on the results.
```